Documentation/UserTsconfig/Setup.rst

```diff
@@ -98,7 +98,8 @@ edit_docModuleUpload
     boolean
 
 :aspect:`Description`
-    File upload directly in Doc. module
+    Allow file upload directly from within CE edit masks.
```

Comment:
```suggestion
    Allow file upload directly from within edit masks of content elements.
```

Maybe "forms" or "backend forms" instead of "edit masks"?